### PR TITLE
[launcher] Update image signature in test

### PR DESCRIPTION
**Pull request by jkl73**
Based on the container workload change in #413

https://pantheon.corp.google.com/artifacts/docker/confidential-space-images-dev/us/cs-cosign-tests/oda/sha256:52aa82f04007625b838ab88a11bfa7ec62baff69c1ad72c46e5acf46d5549c3d?e=13802955&mods=allow_workbench_image_override
https://pantheon.corp.google.com/artifacts/docker/confidential-space-images-dev/us-west1/cs-integ-test-images/ipc%2Fhappypath/sha256:a7d9b216e16ad1fb2b1e8a35e3da58b21ee8dba84c3b4970567d7ec0234a4010?e=13802955&mods=allow_workbench_image_override

### CODE REVIEW

File: launcher/image/test/test_oda_with_signed_container.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ substitutions:
   '_VM_NAME_PREFIX': 'oda-signedcontainer'
   '_ZONE': 'us-east1-b'
   # If the workload image changes, the commit author should change the cosign signature as well to not break tests.
-  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/ipc/happypath@sha256:999831a7b8f8afd323e2359f3c1192206be2aa1d4f3b19f0739eff5f01f83b9e'
+  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/ipc/happypath@sha256:a7d9b216e16ad1fb2b1e8a35e3da58b21ee8dba84c3b4970567d7ec0234a4010'
   '_SIGNATURE_REPO': 'us-docker.pkg.dev/confidential-space-images-dev/cs-cosign-tests/oda'
   '_EXPECTED_SIG': 'EQCIBIEAGtAqxzhMqq2jhs76KJZaD5VXvKV93yotPUtD7yDAiAFS4zFyiRicrhzeS1nmb9SpuAtDRSwv1lWaSWxWiwLTw=='
```
Comment (yawangwang) [inline launcher/image/test/test_oda_with_signed_container.yaml:10]: Please update the container signature to let the test pass.

File: launcher/image/test/test_oda_with_signed_container.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ substitutions:
   '_VM_NAME_PREFIX': 'oda-signedcontainer'
   '_ZONE': 'us-east1-b'
   # If the workload image changes, the commit author should change the cosign signature as well to not break tests.
-  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/ipc/happypath@sha256:999831a7b8f8afd323e2359f3c1192206be2aa1d4f3b19f0739eff5f01f83b9e'
+  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/ipc/happypath@sha256:a7d9b216e16ad1fb2b1e8a35e3da58b21ee8dba84c3b4970567d7ec0234a4010'
```
Comment (alexmwu) [inline launcher/image/test/test_oda_with_signed_container.yaml:8]: Please add the commands to update the sig and point to this commit hash in https://github.com/google/go-tpm-tools/blob/main/launcher/image/testworkloads/customtoken/happypath/Dockerfile

File: launcher/image/test/test_oda_with_signed_container.yaml (hunk 1 above)
Comment (jkl73) [reply launcher/image/test/test_oda_with_signed_container.yaml]: Done

File: launcher/image/test/test_oda_with_signed_container.yaml (hunk 2 above)
Comment (jkl73) [reply launcher/image/test/test_oda_with_signed_container.yaml:8]: Done

**Review by alexmwu — APPROVED** [review]

**Review by yawangwang — APPROVED** [review]

**Pull request merged**